app/src/main/res/raw/buildingcoordinates.json

```diff
@@ -15,21 +15,24 @@
         "type": "Polygon",
         "coordinates": [
```

Comment:
Buildingcoordinates is not used anymore. It will be deleted soon

Reply:
ok..